src/style.css

```diff
@@ -1,3 +1,78 @@
-body {
+:root {
+  --nav-active: #00acdc;
+  --header-color: #fff;
+  --nav-color: #000;
+  --nav-backgroundcolor: #fff;
+}
+
+* {
```

Comment:
better not adding a lot of styles to * selector without necessary

For example, if you add font-family and font-size to `body` they will be inherited by all nested elements
It is not a good idea to reset the margin for all elements, better to do it for the list of tags you really need

What really should be here is `box-sizing` property